great_expectations/core/validation_definition.py

```diff
@@ -273,3 +277,15 @@ def identifier_bundle(self) -> _IdentifierBundle:
         self.suite.identifier_bundle()
 
         return _IdentifierBundle(name=self.name, id=self.id)
+
+    @public_api
+    def save(self) -> None:
```

Comment:
Surprised we missed this one - I'm imagining we want this to ensure parity across domain objects right?

Reply:
Oh weird. Yeah, definitely seems like we want it here.